.github/workflows/testing.yml

```diff
@@ -111,6 +111,9 @@ jobs:
           more /tmp/sequencer.out
           diff -u /tmp/sequencer.out etc/sequencer.out
           diff -u /tmp/generated.md docs/specs/sequences/generated.md
+          ls -1 validator/sequences | xargs -I% diff -u \
```

Comment:
be warned, what I discovered the other day is that the shell in GitHub Actions doesn't always process failure conditions correctly. Did you check that this actually fails the run when there's a problem? ( You could quickly check by tweaking the validator/sequences/%/sequence.md file)

Reply:
Thanks, I've checked and it does fail when there is a difference or missing file.